Sprint-2/implement/to-pounds.js

```diff
@@ -4,3 +4,21 @@
 // You will need to declare a function called toPounds with an appropriately named parameter.
 
 // You should call this function a number of times to check it works for different inputs
+function toPounds(penseString) {
+    const penceStringWithoutTrailingP = penseString.slice(
+      0,
+      penseString.length - 1
+    );
+    const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+    const pounds = paddedPenceNumberString.slice(
+      0,
+      paddedPenceNumberString.length - 2
+    );
+  
+    const pense = paddedPenceNumberString.slice(
+      paddedPenceNumberString.length - 2
+    );
+    return `£${pounds}.${pense}`;
+  }
+  
+  console.log(toPounds("100p"));
```

Comment:
```
// You should call this function a number of times to check it works for different inputs
```

To help you address all of the requirements for this task, I would suggest adding more than one `console.log` to show you've tested multiple inputs!